### ISSUE TITLE
Meta: Games that have broken lens flares

### ISSUE THREAD

**Issue by TheTechnician27** [PCSX2/pcsx2#10643]
### Describe the Bug

This thread will be for games whose lens flares are broken in some version of PCSX2 at or beyond nightly v1.7.5450. Hopefully accounting for all possibilities:

- Games whose lens flares have, at some point past 5450, been made to work on default global settings will be checked.
- Games which currently require a GameDB entry to fix will be checked off but will be distinguished with an asterisk (*).
- Games which currently require some sort of hack (e.g. "Manual Hardware Renderer Fixes") will be checked off but will be distinguished with a dagger (†).
- Games whose only known fix is a patch will be unchecked and will be denoted with a lozenge (◊). (I'm 99% sure this isn't a thing, but I'm covering my bases.)
- Games which are not currently fixed whatsoever will be unchecked.

Broken lens flares can include: not showing up at all or in specific instances when they are intended to, showing up when they're not supposed to, and displaying but incorrectly. Although this is primarily targeted toward hardware mode, if it (verifiably) does not work on software mode, please post it here. This seems like a niche problem on the surface, but this is apparently quite widespread when it comes to games with lens flare effects. If you or a loved one has suffered from problems with lens flare, please post it here, preferably with a GS dump but at minimum with a screenshot comparison.

Hoping to collate these here because 1) it may make it easier to find commonalities, 2) if a GameDB change does work for one, we can see if it works for others, 3) it might minimize the number of new issues in the future (searching "flare" under issues yields 14 results), and importantly 4) I feel like this sort of issue probably slips through the cracks a lot, and this will hopefully bring some attention to it.

- [ ] Spider-Man: The Movie (will be fixed via GameDB by #10641 )

### Reproduction Steps

This is filler

### PCSX2 Revision

v1.7.5450

### Operating System

Linux (64bit) - Specify distro below

### CPU

5800XX

### GPU

1070

**Comment by MrCK1** [comment]
I don't think this warrants a meta issue, likely none of the broken games are using the same rendering techniques. I would stick to individual issues.

**Comment by TheTechnician27** [comment]
> I don't think this warrants a meta issue, likely none of the broken games are using the same rendering techniques. I would stick to individual issues.

From what I've heard from Jordan, there are games where the same GameDB fix (specifically GPU CLUT) fixes multiple games. There are probably still quite a few games where this is an issue, and even if they don't all stem from the same source, I think there have been more than enough occurrences that a meta issue is worthwhile to keep them centralized.
